src/actions/PS2AlertsMetagameInstanceStartAction.ts

```diff
@@ -0,0 +1,83 @@
+import PS2AlertsMetagameInstance from '../instances/PS2AlertsMetagameInstance';
+import {getLogger} from '../logger';
+import {ActionInterface} from '../interfaces/ActionInterface';
+import MongooseModelFactory from '../factories/MongooseModelFactory';
+import Census from '../config/census';
+import {rest} from 'ps2census';
+import {InstanceFacilityControlSchemaInterface} from '../models/instance/InstanceFacilityControlModel';
+
+export default class PS2AlertsMetagameInstanceStartAction implements ActionInterface {
+    private static readonly logger = getLogger('PS2AlertsMetagameInstanceStartAction');
+    private readonly instance: PS2AlertsMetagameInstance;
+    private readonly instanceFacilityControlModelFactory: MongooseModelFactory<InstanceFacilityControlSchemaInterface>;
+    private readonly censusConfig: Census;
+
+    constructor(
+        instance: PS2AlertsMetagameInstance,
+        instanceFacilityControlModelFactory: MongooseModelFactory<InstanceFacilityControlSchemaInterface>,
+        censusConfig: Census,
+    ) {
+        this.instance = instance;
+        this.instanceFacilityControlModelFactory = instanceFacilityControlModelFactory;
+        this.censusConfig = censusConfig;
+    }
+
+    public async execute(): Promise<boolean> {
+        PS2AlertsMetagameInstanceStartAction.logger.info(`Running startActions() for instance "${this.instance.world}-${this.instance.censusInstanceId}"`);
+        // Take a snapshot of the map for use with territory calculations for the end
+
+        const get = rest.getFactory('ps2', this.censusConfig.serviceID);
+        const docs: any[] = [];
+
+        await get(
+            rest.join(
+                rest.map,
+                [{
+                    type: 'map_region',
+                    // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Maybe disable this rule for the file? Or maybe scrap it all together? So far this rule has not been very helpfull and only annoying af.

Reply:
Yeah, will consider it, not for now though :)